pkg/cmd/api/api.go

```diff
@@ -219,7 +219,7 @@ func NewCmdApi(f *cmdutil.Factory, runF func(*ApiOptions) error) *cobra.Command
 	cmd.Flags().BoolVar(&opts.Paginate, "paginate", false, "Make additional HTTP requests to fetch all pages of results")
 	cmd.Flags().StringVar(&opts.RequestInputFile, "input", "", "The `file` to use as body for the HTTP request (use \"-\" to read from standard input)")
 	cmd.Flags().BoolVar(&opts.Silent, "silent", false, "Do not print the response body")
-	cmd.Flags().StringVarP(&opts.Template, "template", "t", "", "Format the response using a Go template")
+	cmd.Flags().StringVarP(&opts.Template, "template", "t", "", "Format JSON output using a Go template; see \"gh help formatting\"")
```

Comment:
This line of docs gets automatically added to any command that has a `--template` flag:

```
For more information about output formatting flags, see `gh help formatting`.
```

Do you think it's also worth pointing this out in the description of the flag itself?

Reply:
Personally, not much. But see the linked bug for the discussion that lead to this PR.

Reply:
Ping